src/import/prepare.rs

```diff
@@ -180,6 +183,23 @@ impl<Client: ImportClient> PrepareRangeJob<Client> {
         }
         match self.split_region(&region) {
             Ok(new_region) => {
+                // We need to wait for a few milliseconds, because PD may have
```

Comment:
Actually, the scatter region request will take the new split region info, so we can just adjust PD logic to update the region info when receiving scatter requests so that no need to wait here.

Reply:
Hmm could this logic be implemented in 2.1.6?

Reply:
seems the change won't too much

Reply:
help you change that, PTAL https://github.com/pingcap/pd/pull/1457

Reply:
Thanks. Will close this after 1457 is merged.

Reply:
the proposal is not accepted by PD. So still using this way.